src/SeqCli/Config/SeqCliConnectionConfig.cs

```diff
@@ -23,6 +23,9 @@ class SeqCliConnectionConfig
     const string ProtectedDataPrefix = "pd.";
 
     public string ServerUrl { get; set; } = "http://localhost:5341";
+    public ulong EventBodyLimitBytes { get; set; } = 256 * 1024;
+    public ulong RawPayloadLimitBytes { get; set; } = 10 * 1024 * 1024;
+    public uint? PooledConnectionLifetimeMilliseconds { get; set; } = null;
```

Comment:
I'd be tempted to move `PooledConnectionLifetimeMilliseconds` under a `fowarder.connection` section, too.

If we put `EventBodyLimitBytes` and `RawPayloadLimitBytes` at this level, could we apply them as defaults to the other output commands?